docs/visual-basic/language-reference/error-messages/unable-to-emit-assembly-error-message.md

```diff
@@ -24,15 +24,15 @@ The [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] compiler calls the Assembly Link
   
 ## To correct this error  
   
-1.  Examine the quoted error message and consult the topic [Al.exe Tool Errors and Warnings](http://msdn.microsoft.com/en-us/7f125d49-0a03-47a6-9ba9-d61a679a7d4b) for further explanation and advice.  
+1.  Examine the quoted error message and consult the topic [Al.exe](../../../framework/tools/al-exe-assembly-linker.md). for further explanation and advice.  
   
-2.  Try signing the assembly manually, using either the [Al.exe (Assembly Linker)](https://msdn.microsoft.com/library/c405shex) or the [Sn.exe (Strong Name Tool)](https://msdn.microsoft.com/library/k5b5tt23).  
+2.  Try signing the assembly manually, using either theor the [Sn.exe (Strong Name Tool)][Sn.exe (Strong Name Tool)](../../../framework/tools/sn-exe-strong-name-tool.md)).  
```

Comment:
typo introduced theor, the AL link should go to https://docs.microsoft.com/en-us/dotnet/framework/tools/al-exe-assembly-linker instead of being removed